cli/src/config/templates.toml

```diff
@@ -66,6 +66,8 @@ separate(" ",
 )
 '''
 
+duplicate_commit_description = ''
```

Comment:
We probably don't want to add a default, but perhaps an alias called `cherry_pick_commit_description` or something would be good; then we can reference it in the Git command table.

Reply:
Hm, should this be named `duplicate_description` instead, like `revert_description`?